cve_bin_tool/csv2cve.py

```diff
@@ -70,8 +71,14 @@ def csv2cve(filename):
             )
             cves = cvedb.get_cves(row["vendor"], row["product"], row["version"])
             if cves:
-                LOGGER.info("\n".join(sorted(cves.keys())))
-                cveoutput.append(cves.keys())
+                s = map(
+                    lambda key: " ".join(
+                        [row["vendor"], row["product"], row["version"], key, cves[key]]
+                    ),
+                    sorted(cves.keys()),
+                )
+                LOGGER.info("\n".join(s))
+                cveoutput.append(cves)
```

Comment:
I was going to suggest that we switch this over to use OutputEngine, but this is a lovely interim solution until we get to the point of deduping our output, thank you.